### ISSUE TITLE
Journal Entry Page is Not Styled when Opening Just that Page

### ISSUE THREAD

**Issue by Larkinabout** [foundryvtt/dnd5e#4288]
When opening just the Journal Entry Page of a document, the content does not pick up the dnd5e style. Additionally, the `journal-page-header` is `flex: 1` due to `.window-app .window-content > *` and there is no padding around `journal-page-header` nor `journal-page-content`.
![image](https://github.com/user-attachments/assets/7b8a9ce2-2657-44dd-86dd-3971c7d37eb1)

**Comment by arbron** [comment]
This seems to be a module opening a journal page in a non-standard way, standard method for showing a single journal page to players results in it still opening within a full journal entry application.

**Comment by arbron** [comment]
I see, you are rendering the journal entry page sheet directly.

**Comment by Larkinabout** [comment]
Yes, I thought it was a nice way of just showing the specific content I want the user to see rather than within the full journal. As it worked, I thought it was a purposeful, intended method, but just missing the styling. Specifically, I grab the document  from the UUID of the entry using `fromUuid`, then render the sheet.

Realise now that the lack of padding is most likely because it's not actually fully supported with core Foundry either.

**Comment by arbron** [comment]
> Yes, I thought it was a nice way of just showing the specific content I want the user to see rather than within the full journal. As it worked, I thought it was a purposeful, intended method, but just missing the styling. Specifically, I grab the document from the UUID of the entry using `fromUuid`, then render the sheet.
> 
> Realise now that the lack of padding is most likely because it's not actually fully supported with core Foundry either.

Yeah, probably not the way to go about it. You could always set it up to use a rich tooltip and point them towards rule pages.

**Comment by Larkinabout** [comment]
Thanks, I'll try that.

I'll also pop an issue over on Foundry VTT side as I think it'd be a nice option and it's almost working as is.
